automation/vars/default.yaml

```diff
@@ -0,0 +1,56 @@
+---
+vas:
+  hci:
+    repo: https://github.com/openstack-k8s-operators/architecture.git
+    branch: main
+    stages:
+      - path: examples/va/hci/control-plane/nncp
+        validations:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=60
```

Comment:
ah, you missed the `s` for the time unit :).

Reply:
Yes, that is my fault for an initial bad copy-paste.  Will submit a PR.